staging/src/github.com/kubeedge/mapper-framework/_template/mapper/data/dbmethod/tdengine/client.go

```diff
@@ -0,0 +1,163 @@
+package tdengine
+
+import (
+	"database/sql"
+	"encoding/json"
+	"fmt"
+	"github.com/kubeedge/mapper-generator/pkg/common"
+	_ "github.com/taosdata/driver-go/v3/taosRestful"
+	"k8s.io/klog/v2"
+	"os"
+	"strings"
+	"time"
+)
+
+var (
+	DB *sql.DB
+)
+
+type DataBaseConfig struct {
+	Config *ConfigData `json:"config,omitempty"`
+}
+type ConfigData struct {
```

Comment:
change configdata to tdengineClientConfig as what we show in device_instance.yaml may be better